servers/src/main/java/tachyon/worker/block/SpaceReserver.java

```diff
@@ -52,15 +54,25 @@
   public SpaceReserver(BlockDataManager blockManager) {
     mBlockManager = blockManager;
     mStorageTierAssoc = new WorkerStorageTierAssoc(WorkerContext.getConf());
-    Map<String, Long> capOnTiers = blockManager.getStoreMeta().getCapacityBytesOnTiers();
+    Map<String, Long> capOnTiers = mBlockManager.getStoreMeta().getCapacityBytesOnTiers();
     long lastTierReservedBytes = 0;
     for (int ordinal = 0; ordinal < mStorageTierAssoc.size(); ordinal ++) {
-      String tierReservedSpaceProp =
-          String.format(Constants.WORKER_TIERED_STORE_LEVEL_RESERVED_RATIO_FORMAT, ordinal);
       String tierAlias = mStorageTierAssoc.getAlias(ordinal);
-      long reservedSpaceBytes =
+      // To-do: make sure highWatermark ratio is larger than lowWatermark ratio
+      // HighWatemark defines when to start the space reserving process
+      String highWatermarkRatio =
+          String.format(Constants.WORKER_TIERED_STORE_LEVEL_HIGH_WATERMARK_RATIO_FORMAT, ordinal);
+      long highWatermarkInBytes =
           (long) (capOnTiers.get(tierAlias) * WorkerContext.getConf().getDouble(
-              tierReservedSpaceProp));
+              highWatermarkRatio));
+      mHighWaterMarkInBytesOnTiers.put(tierAlias, highWatermarkInBytes);
+
+      // LowWatemark defines when to stop the space reserving process if started
+      String lowWatermarkRatio =
+          String.format(Constants.WORKER_TIERED_STORE_LEVEL_LOW_WATERMARK_RATIO_FORMAT, ordinal);
+      long reservedSpaceBytes =
+          (long) (capOnTiers.get(tierAlias)
+              * (1 - WorkerContext.getConf().getDouble(lowWatermarkRatio)));
       mBytesToReserveOnTiers.put(tierAlias, reservedSpaceBytes + lastTierReservedBytes);
```

Comment:
Maybe change this to match `mHigherWaterMarkInBytesOnTiers`? Or is it used in other cases as well?